internal/commands/document.go

```diff
@@ -92,7 +99,16 @@ func runDocCommand(path string) error {
 		return fmt.Errorf("get documentation: %w", err)
 	}
 
-	t, err := template.New("docs").Parse(docTemplate)
+	if viper.GetString("template-file") != "" {
```

Comment:
nit: Consider the following so `GetString` is only called once

```go
if file := viper.GetString("template-file"); file != "" {
...
```

Reply:
Thanks, fixed!